CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
-## 0.0.3 UNRELEASED
+## 0.1.0
 
 FEATURES:
 
 * config: Add ability to configure container network_mode [[GH-33](https://github.com/hashicorp/nomad-driver-podman/issues/33)]
 * network: (Consul Connect) Ability to accept a bridge network namespace from Nomad. [[GH-38](https://github.com/hashicorp/nomad-driver-podman/issues/38)]
+* runtime: Ability to run podman rootless [[GH-42](https://github.com/hashicorp/nomad-driver-podman/issues/42)]
+* config: Ability to specify varlink socket path [[GH-42](https://github.com/hashicorp/nomad-driver-podman/issues/42)]
+* runtime: Conditionally set memory swappiness only if cgroupv1 is running [[GH-42](https://github.com/hashicorp/nomad-driver-podman/issues/42)]
```

Comment:
just double checking all 3 of these are GH-42 ?

Reply:
yeah, big pr :sweat_smile: